docs/features.md

```diff
@@ -1,21 +1,51 @@
-- ✅ Builtin [Fira Code inspired progress bar](https://github.com/contour-terminal/contour/issues/521) support.
-- ✅ Read-only mode, protecting against accidental user-input to the running application, such as <kbd>Ctrl</kbd>+<kbd>C</kbd>.
-- ✅ and much more ...
+:material-check-bold:{.check-mark}  Available on all 4 major platforms, Linux, OS/X, FreeBSD, Windows.
 
+:material-check-bold:{.check-mark}  GPU-accelerated rendering.
+
+:material-check-bold:{.check-mark}  [Font ligatures](demo/font-ligatures.md) support (very useful for programming and scripting)
+
+:material-check-bold:{.check-mark}  Unicode: Emoji support (-: 🌈 💝 😛 👪 - including ZWJ, VS15, VS16 emoji :-)
+
+:material-check-bold:{.check-mark}  Unicode: Grapheme cluster support
+
+:material-check-bold:{.check-mark}  Bold and italic fonts
+
+:material-check-bold:{.check-mark}  High-DPI support.
+
+:material-check-bold:{.check-mark}  [Vi-like input modes](input-modes.md) for improved navigation, selection and copy'n'paste experience
+
+:material-check-bold:{.check-mark}  [Vertical Line Markers](demo/line-marks.md) (quickly jump to markers in your history!)
+
+:material-check-bold:{.check-mark}  Blurred behind transparent background when using Windows 10 or KDE window manager on Linux.
+
+:material-check-bold:{.check-mark}  Blurrable Background image support.
+
+:material-check-bold:{.check-mark}  Runtime configuration reload
+
+:material-check-bold:{.check-mark}  256-color and Truecolor support
+
+:material-check-bold:{.check-mark}  Key binding customization
+
+:material-check-bold:{.check-mark}  Color Schemes
+
+:material-check-bold:{.check-mark}  Profiles (grouped customization of: color scheme, login shell, and related behaviours)
+
+:material-check-bold:{.check-mark}  [Synchronized rendering](https://github.com/contour-terminal/contour/wiki/VTExtensions#synchronized-output) (via `SM ? 2026` / `RM ? 2026`)
+
+:material-check-bold:{.check-mark}  Text reflow (configurable via `SM ? 2028` / `RM ? 2028`)
+
+:material-check-bold:{.check-mark}  Clickable hyperlinks via [OSC 8](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda)
+
+:material-check-bold:{.check-mark}  Clipboard setting via OSC 52
+
+:material-check-bold:{.check-mark}  [Sixel Image support](demo/images.md)
+
+:material-check-bold:{.check-mark}  Terminal page [buffer capture VT extension](https://github.com/contour-terminal/contour/wiki/VTExtensions#buffer-capture) to quickly extract contents.
+
+:material-check-bold:{.check-mark}  Builtin [Fira Code inspired progress bar](https://github.com/contour-terminal/contour/issues/521) support.
+
+:material-check-bold:{.check-mark}  Read-only mode, protecting against accidental user-input to the running application, such as <kbd>Ctrl</kbd>+<kbd>C</kbd>.
+
+:material-check-bold:{.check-mark}  [VT320 Host-programmable and Indicator statusline support](demo/statusline.md)
```

Comment:
instead of empty lines (which makes each line being a paragraph and therefore a little bit to space, try without empty lines but with a `<br/>` at the end of each line (weird, i know, but that works at least (tried locally))

```suggestion
:material-check-bold:{.check-mark}  Available on all 4 major platforms, Linux, OS/X, FreeBSD, Windows. <br/>
:material-check-bold:{.check-mark}  GPU-accelerated rendering. <br/>
:material-check-bold:{.check-mark}  [Font ligatures](demo/font-ligatures.md) support (very useful for programming and scripting) <br/>
:material-check-bold:{.check-mark}  Unicode: Emoji support (-: 🌈  💝  😛  👪  - including ZWJ, VS15, VS16 emoji :-) <br/>
:material-check-bold:{.check-mark}  Unicode: Grapheme cluster support <br/>
:material-check-bold:{.check-mark}  Bold and italic fonts <br/>
:material-check-bold:{.check-mark}  High-DPI support. <br/>
:material-check-bold:{.check-mark}  [Vi-like input modes](input-modes.md) for improved navigation, selection and copy'n'paste experience <br/>
:material-check-bold:{.check-mark}  [Vertical Line Markers](demo/line-marks.md) (quickly jump to markers in your history!) <br/>
:material-check-bold:{.check-mark}  Blurred behind transparent background when using Windows 10 or KDE window manager on Linux. <br/>
:material-check-bold:{.check-mark}  Blurrable Background image support. <br/>
:material-check-bold:{.check-mark}  Runtime configuration reload <br/>
:material-check-bold:{.check-mark}  256-color and Truecolor support <br/>
:material-check-bold:{.check-mark}  Key binding customization <br/>
:material-check-bold:{.check-mark}  Color Schemes <br/>
:material-check-bold:{.check-mark}  Profiles (grouped customization of: color scheme, login shell, and related behaviours) <br/>
:material-check-bold:{.check-mark}  [Synchronized rendering](https://github.com/contour-terminal/contour/wiki/VTExtensions#synchronized-output) (via `SM ? 2026` / `RM ? 2026`) <br/>
:material-check-bold:{.check-mark}  Text reflow (configurable via `SM ? 2028` / `RM ? 2028`) <br/>
:material-check-bold:{.check-mark}  Clickable hyperlinks via [OSC 8](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) <br/>
:material-check-bold:{.check-mark}  Clipboard setting via OSC 52 <br/>
:material-check-bold:{.check-mark}  [Sixel Image support](demo/images.md) <br/>
:material-check-bold:{.check-mark}  Terminal page [buffer capture VT extension](https://github.com/contour-terminal/contour/wiki/VTExtensions#buffer-capture) to quickly extract contents. <br/>
:material-check-bold:{.check-mark}  Builtin [Fira Code inspired progress bar](https://github.com/contour-terminal/contour/issues/521) support. <br/>
:material-check-bold:{.check-mark}  Read-only mode, protecting against accidental user-input to the running application, such as <kbd>Ctrl</kbd>+<kbd>C</kbd>. <br/>
:material-check-bold:{.check-mark}  [VT320 Host-programmable and Indicator statusline support](demo/statusline.md) <br/>
```